seismic/receiver_fn/config_moho_workflow_example.json

```diff
@@ -0,0 +1,26 @@
+{
+    "methods":
+    {
+        "CCP_1":
+        {
+            "csv_file": "sandbox/PV-227/ccp_line_data_sample1.csv",
+            "weighting": 1.0,
+            "scale_length_degrees": 0.2,
+            "enable_sample_weighting": false
+        },
+        "CCP_2":
+        {
+            "csv_file": "sandbox/PV-227/ccp_line_data_sample2.csv",
+            "weighting": 0.5,
```

Comment:
This version of JSON schema looks much better

Reply:
Thank you Fei for the review